phylogenetic/rules/export.smk

```diff
@@ -70,3 +70,22 @@ rule export:
             --output {output.auspice} \
             --include-root-sequence-inline
         """
+
+rule final_strain_name:
+    input:
+        auspice_json="results/lassa_{segment}.json",
+        metadata="data/metadata_{segment}.tsv",
+    output:
+        auspice_json="auspice/lassa_{segment}.json",
+    params:
+        strain_id_field=config["strain_id_field"],
+        display_strain_field=config["display_strain_field"],
+    shell:
+        """
+        python3 scripts/set_final_strain_name.py \
```

Comment:
_non-blocking_
Another reminder for us to get back to https://github.com/nextstrain/auspice/pull/1668. 

Is setting the strain name helpful for lassa? It looks like a majority of the strain names are the GenBank accession anyways.

Reply:
In practice, strain names are commonly used on phylogenetic trees for two main reasons:

* Communication: It's easier to discuss and refer to a named strain rather than a complex identifier.
* Standardization: Using strain names encourages sample submitters to develop clear naming conventions.

For Lassa virus specifically, the strain name serves an additional important function. Since researchers can submit two separate GenBank samples per strain (one for the L segment and one for the S segment), the consistent use of strain names allows looking at tanglegrams of the segments.

Thanks for flagging! In my quick exploration, indeed there was only [49% of the samples getting strain names](https://github.com/j23414/generated-reports/blob/main/reports/lassa.pdf). After digging in, [rescued several more](https://github.com/nextstrain/lassa/pull/11) so that percentage should be higher.